ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/DeleteItemAction.java

```diff
@@ -173,7 +173,7 @@ protected void onSuccess(final Status result) {
 
                     @Override
                     protected void onFailure(Throwable exception) {
-                        notificationManager.showError(exception.getMessage());
+                        notificationManager.notify("Git", exception.getMessage());
```

Comment:
Because we got failed response from git status operation.

"Git" - its title for the notification.